tools/ci/ci-packages.txt

```diff
@@ -198,7 +198,7 @@ perl-Module-Implementation-0.09
 perl-Module-Pluggable-5.2
 perl-Module-Runtime-0.016
 perl-Module-Runtime-Conflicts-0.003
-perl-Mojo-IOLoop-ReadWriteProcess-0.340.0
+perl-Mojo-IOLoop-ReadWriteProcess-0.33
```

Comment:
That doesn't seem correct, why is it downgrading?
We have 0.340 in Factory: https://build.opensuse.org/package/show/devel:languages:perl/perl-Mojo-IOLoop-ReadWriteProcess

Reply:
Fixed in https://github.com/os-autoinst/openQA/pull/5355